src/main/java/pl/nogacz/snake/board/Board.java

```diff
@@ -178,13 +255,346 @@ public void readKeyboard(KeyEvent event) {
+                    }
+
+                    @Override
+                    public void mouseExited(java.awt.event.MouseEvent e) {
+                        // TODO Auto-generated method stub
+                    }
+                });
+
+                confirm.addActionListener(new ActionListener() {
+
+                    @Override
+                    public void actionPerformed(java.awt.event.ActionEvent e) {
+
+                        String adress = input.getText();
+
+                        if (adress.equals(""))
+                            input.setText("Please enter a destination adress");
+
+                        else
+                            cancel=false;
+
+                            if(startLoad(adress)){
+
+                                getMessage(1, 0);                              
+
+                            }
+
+                            else
+                                getMessage(1, 1);
+
+                                frame.dispose();
+
+                    }
+                });
+
+                frame.setSize(500, 125);
+                frame.setDefaultCloseOperation(WindowConstants.DISPOSE_ON_CLOSE);
+                frame.addWindowListener(new WindowListener() {
+                @Override
+                public void windowClosing(java.awt.event.WindowEvent we) {
+                    
+                }
+
+				@Override
+				public void windowOpened(WindowEvent e) {
+					// TODO Auto-generated method stub
+					
+				}
+
+				@Override
+				public void windowClosed(WindowEvent e) {
+                    if(cancel)
+                        getMessage(1, -1);
+					
+				}
+
+				@Override
+				public void windowIconified(WindowEvent e) {
+					// TODO Auto-generated method stub
+					
+				}
+
+				@Override
+				public void windowDeiconified(WindowEvent e) {
+					// TODO Auto-generated method stub
+					
+				}
+
+				@Override
+				public void windowActivated(WindowEvent e) {
+					// TODO Auto-generated method stub
+					
+				}
+
+				@Override
+				public void windowDeactivated(WindowEvent e) {
+					// TODO Auto-generated method stub
+					
+				}
+
+                });
+
+                input.setPreferredSize(new Dimension(400, 25));
+                frame.toFront();
+
+                panel.setPreferredSize(new Dimension(400, 400)); 
+
+                panel.add(label);
+                panel.add(input);
+                panel.add(confirm);
+                
+                frame.add(panel);
+
+                frame.setVisible(true);
+
+                break;
+
+            default:
+                    break;
+        }
+    }
+
+    private boolean startLoad(String adress){
+
+        FileInputStream fi;
+        ObjectInputStream in;
+
+        try {
+
+            fi=new FileInputStream(adress);
+            in=new ObjectInputStream(fi);
+
+            BoardInfo BI=(BoardInfo)in.readObject();
+
+            in.close();
+            fi.close();
+
+            this.board=BI.getBoard();

```

Comment:
Yaptığın değişikliklerin genelinde spacing'e dikkat. Yaptığın değişiklikler kodda genel olarak olan spacing ile uyuşmuyor.

`this.board=BI.getBoard()` => `this.board = BI.getBoard()`

şeklinde her parçadan sonra birer bir boşluk bırak.

Reply:
Done.